LearnosityDemo/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+
+namespace LearnosityDemo.Pages
+{
+    public class IndexModel : PageModel
+    {
+        private readonly ILogger<IndexModel> _logger;
+
+        public IndexModel(ILogger<IndexModel> logger)
+        {
+            _logger = logger;
+        }
+
+        public void OnGet()
```

Comment:
try removing this `OnGet` function and see if anything breaks - if it doesn't do anything then we should remove it.

Reply:
Runs fine without, so I deleted it. https://github.com/Learnosity/learnosity-sdk-asp.net/commit/4ee05704adf4d448d380f2038232e064e80762d3